### ISSUE TITLE
Think as OSPARC, Sim4Life Web, TIP as databases

### ISSUE THREAD

**Issue by AntoninoMarioC** [ITISFoundation/osparc-issues#887]
dear all,
As OSPARC is now, it is not just a platform to develop pipelines or to run simulations (e.g. Sim4Life web or TIP), but it is **also a database**. In facts, projects and pipelines can be stored remotely, and the system of tagging projects is in my opinion not sufficient. As well, at the moment, the projects are sorted by the date of opening, and this is not very ideal.

We would need to identify minimal requirements to handle a database of projects from the dashboard. The first, in my opinion, should be the **possibility to create of folders and subfolders** (e.g. Project A, Project B, etc.) with the following requirements:
1. possibility to share whole folders with partners
2. authorization
3. sorting
4. hiding from the dashboard (e.g. I do not want to show some folders to my guests)
5. zipping
6. tree & database structure
7. all the rest - am sure you know what's needed more than me!

It would be great that we discuss the priority of this case soon, before that the full Sim4Life Web is released. Am pretty sure that big users of OSPARC (e.g. Taylor, etc). would also appreciate the improvement.

All the best,
Antonino

**Comment by newton1985** [comment]
I agree such a discussion would be useful and would be happy to participate.

**Comment by elisabettai** [comment]
Hi @AntoninoMarioC, thanks for this feedback. Before starting the discussion, could you maybe write down a (a couple of) user stories so we can understand better what you want to achieve? It might be that we already have functionality for your goals.

One user story I can imagine is:

_I have some studies (templates) that I want to share with Group A, some with Group B. When I meet with Group A, I don't want that they see the studies (templates) I shared with Group B_

Feel free to edit, if that's not what you have in mind and add more user stories.
